spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/metrics.adoc

```diff
@@ -169,15 +169,31 @@ The {dynatrace-help}/dynatrace-api/basics/dynatrace-api-authentication/[API toke
 It is recommended to limit the scope of the token to this one permission.
 Please ensure that the endpoint URI contains the path (e.g. `/api/v2/metrics/ingest`).
 
+The URLs of the Metrics API v2 ingest endpoint are:
+
+- `+https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest+` for SaaS and
+- `+https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest+` for Managed Deployments
+
 [source,yaml,indent=0,subs="verbatim",configprops,configblocks]
 ----
 	management:
 	  metrics:
 	    export:
 	      dynatrace:
-	        # uri: "https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest" for managed deployments.
 	        uri: "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest"
-	        api-token: "YOUR_TOKEN" # should be read from a secure source and not hard-coded.
+	        api-token: "YOUR_TOKEN"
+----
+
+Depending on your setup, the environment variables `+DT_METRICS_INGEST_URL+` and `+DT_METRICS_INGEST_API_TOKEN+` might be populated accordingly already and can be used via the Spring placeholder notation:
```

Comment:
What does "depending on your setup" mean? I think that if something is setting those, perhaps we should link to whatever is setting them rather and put that doc there? I am mentioning this as I would prefer that our documentation does not provide too much information for something that would be better covered elsewhere anyway.

Reply:
@snicoll I have removed the example for now since I don't have any good reference to put into the docs. Thanks for pointing out that that might be unclear for readers!